source/appModules/textinputhost.py

```diff
@@ -8,3 +8,4 @@
 
 # Flake8/F403: alias of Composable Shell modern keyboard app module.
 from .windowsinternal_composableshell_experiences_textinput_inputapp import * # NOQA

```

Comment:
Shouldn't there be two spaces before the comment here?
```suggestion
from .windowsinternal_composableshell_experiences_textinput_inputapp import *  # NOQA

```

Reply:
This is not introduced as part of this PR, and since we intent to remove this alias in the next compatibility breaking release I'm not sure we should be fixing anything in these files.